src/components/v2/values/chartValuesDiff/ChartValuesView.component.tsx

```diff
@@ -204,51 +225,197 @@ const RadioWithTippy = (children, isFromCDPipeline: boolean, tippyContent: strin
+                                <h2 data-testid="add_cluster_header" className="fs-16 fw-6 lh-1-43 m-0 title-padding">
+                                    <span className="fw-6 fs-16 cn-9">Git Repository</span>
+                                </h2>
+                                <button
+                                    data-testid="header_close_icon"
+                                    type="button"
+                                    className="dc__transparent flex icon-dim-24"
+                                    onClick={handleCloseButton}
+                                >
+                                    <Close className="icon-dim-24" />
+                                </button>
+                            </div>
+                            <div className="mr-20">
+                                <UserGitRepo
+                                    setRepoURL={handleRepoTextChange}
+                                    setSelectedRepoType={handleRepoTypeChange}
+                                    repoURL={repoURL}
+                                    selectedRepoType={selectedRepoType}
+                                    staleData={staleData}
+                                />
+                            </div>
+                        </div>
+                        <div className="w-100 dc__border-top flex right pb-12 pt-12 pl-20 pr-20 dc__position-fixed dc__position-abs bcn-0 dc__bottom-0">
+                            <button
+                                data-testid="cancel_button"
+                                className="cta cancel h-36 lh-36 mr-10"
+                                type="button"
+                                onClick={handleCloseButton}
+                            >
+                                Cancel
+                            </button>
+                            <button
+                                data-testid="save_cluster_list_button_after_selection"
+                                className="cta h-36 lh-36"
+                                type="button"
+                                onClick={handleSaveButton}
+                            >
+                                Save
+                            </button>
+                        </div>
+                    </Drawer>
+                </div>
+            )}
+            {(gitOpsState && allowedDeploymentTypes.indexOf(DeploymentAppTypes.HELM) !== -1) || (showRepoSelector && window._env_.HIDE_GITOPS_OR_HELM_OPTION) ? (
+                <div className="form__input dashed mt-10 flex" style={{ height: '54px' }}>
+                    <div className="mb-10">
+                        <span>
+                            Commit deployment manifests to
+                            <EditIcon className="icon-dim-16 cursor ml-28 pt-4" onClick={toggleDrawer} />
+                        </span>
+                        <a className="dc__ellipsis-right flex left fs-13 fw-4 lh-20 cursor pb-4" onClick={toggleDrawer}>{`${
```

Comment:
Please simplify this nested ternary to increase readability

Reply:
Resolved